tests/js/common/aql/aql-geo-distance-range.js

```diff
@@ -355,26 +263,15 @@ function BTS_471() {
   };
 }
 
-function BTS_471Suite() {
-  let suite = {};
-  deriveTestSuite(
-    BTS_471(),
-    suite,
-    "BTS_471"
-  );
-  return suite;
-}
-  
-function BTS_470Suite() {
```

Comment:
This is valid because test data are same.
Query in reality also same, but in one written geo_in_range and in another geo_distance